### ISSUE TITLE
Error on run

### ISSUE THREAD

**Issue by pixelkritzel** [jeffijoe/typesync#1]
Tried to use it in a project of mine ...

````
~/project$ typesync --dry                                                                                                                                         
~/.nvm/versions/node/v6.10.0/lib/node_modules/typesync/lib/package-json-file-service.js:7
var existsAsync = util.promisify(fs.exists);
                       ^

TypeError: util.promisify is not a function
    at Object.<anonymous> (~/.nvm/versions/node/v6.10.0/lib/node_modules/typesync/lib/package-json-file-service.js:7:24)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (~/.nvm/versions/node/v6.10.0/lib/node_modules/typesync/lib/cli.js:10:35)
    at Module._compile (module.js:570:32)

````

**Comment by jeffijoe** [comment]
Ah, my bad! Node version might be a bit too high. What version are you on?

**Comment by pixelkritzel** [comment]
6.10

**Comment by jeffijoe** [comment]
A fix will be published shortly, thanks for bringing this up!

**Comment by jeffijoe** [comment]
Released as 0.1.1, thanks for reporting!
